File: Euclid/Game/Room/Entity/RoomPlayer.cs
namespace Euclid.Game
{
    public class RoomPlayer : RoomEntity
    {
        #region Fields

        #endregion

        #region Properties

        public Player Player { get; private set; }

        #endregion

        #region Constructors

        public RoomPlayer(Player player) : base(player)
        {
            Player = player;
            TaskObject = new PlayerTaskObject(player);
        }

        #endregion
    }
}

File: euclid/src/main/java/euclid/game/entity/RoomPlayer.java
package euclid.game.entity;

import euclid.game.Player;
import euclid.game.room.tasks.PlayerTaskObject;

public class RoomPlayer extends RoomEntity {

    private final Player player;

    public RoomPlayer(Player player) {
        super(player);
        this.player = player;
        this.setTaskObject(new PlayerTaskObject(player));
    }

    public Player getPlayer() {
        return player;
    }
}
